ts/ts-bas/src/test/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionIdentitetStyrktGenomTest.java

```diff
@@ -0,0 +1,225 @@
+import static se.inera.intyg.common.ts_bas.v7.codes.RespConstantsV7.IDENTITET_STYRKT_GENOM_ID_KORT_TEXT_ID;
+import static se.inera.intyg.common.ts_bas.v7.codes.RespConstantsV7.IDENTITET_STYRKT_GENOM_KORKORT_TEXT_ID;
+import static se.inera.intyg.common.ts_bas.v7.codes.RespConstantsV7.IDENTITET_STYRKT_GENOM_PASS_TEXT_ID;
+import static se.inera.intyg.common.ts_bas.v7.codes.RespConstantsV7.IDENTITET_STYRKT_GENOM_PERS_KANNEDOM_TEXT_ID;
+import static se.inera.intyg.common.ts_bas.v7.codes.RespConstantsV7.IDENTITET_STYRKT_GENOM_SVAR_ID_2;
+import static se.inera.intyg.common.ts_bas.v7.codes.RespConstantsV7.IDENTITET_STYRKT_GENOM_TEXT_ID;
+
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import se.inera.intyg.common.services.texts.CertificateTextProvider;
+import se.inera.intyg.common.support.facade.builder.CertificateBuilder;
+import se.inera.intyg.common.support.facade.model.CertificateDataElement;
+import se.inera.intyg.common.support.facade.model.config.Layout;
+import se.inera.intyg.common.support.facade.model.config.RadioMultipleCode;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueCode;
+import se.inera.intyg.common.support.facade.testsetup.model.CommonElementTest;
+import se.inera.intyg.common.support.facade.testsetup.model.config.ConfigRadioButtonMultipleCodeTest;
+import se.inera.intyg.common.support.facade.testsetup.model.validation.ValidationMandatoryTest;
+import se.inera.intyg.common.support.facade.testsetup.model.value.ValueCodeTest;
+import se.inera.intyg.common.ts_bas.v7.model.internal.Vardkontakt;
+import se.inera.intyg.common.ts_parent.codes.IdKontrollKod;
+
+@ExtendWith(MockitoExtension.class)
+class QuestionIdentitetStyrktGenomTest {
+
+    @Mock
+    private CertificateTextProvider textProvider;
+
+    @BeforeEach
+    void setup() {
+        when(textProvider.get(any(String.class))).thenReturn("Test string");
+    }
+
+    @Nested
+    class ToCertificate {
+
+        @Nested
+        class IncludeCommonElementTest extends CommonElementTest {
+
+            @Override
+            protected CertificateDataElement getElement() {
+                return QuestionIdentitetStyrktGenom.toCertificate(null, 0, textProvider);
+            }
+
+            @Override
+            protected String getId() {
+                return IDENTITET_STYRKT_GENOM_SVAR_ID_2;
+            }
+
+            @Override
+            protected String getParent() {
+                return IDENTITET_CATEGORY_ID;
+            }
+
+            @Override
+            protected int getIndex() {
+                return 0;
+            }
+        }
+
+        @Nested
+        @TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
Regarding my comment above related to this testFixture, this is the reason I think we should avoid using parameterized tests outside of ToInternal, since it might be confusing when to use the @TestInstance(TestInstance.Lifecycle.PER_CLASS) flag or not. Would you agree?